packages/repo-doctor/src/rules/index.js

```diff
@@ -9,16 +9,19 @@ const ruleFns = { depUpToDate, noForbiddenDep, localesInRepo }
 const setupRules = (config, args) => {
   const rules = config.rules.map(rule => {
     let ruleName
-    let options
+    let ruleConfig
     if (Array.isArray(rule)) {
       ruleName = rule[0]
-      options = rule[1]
+      ruleConfig = rule[1]
     } else {
       ruleName = rule
-      options = {}
+      ruleConfig = {}
     }
-    const ruleFn = ruleFns[ruleName]
-    return ruleFn(options, args)
+    let ruleFn = ruleFns[ruleName]
+
+    return ruleFn.prototype
```

Comment:
Maybe we could use `isFunction` instead? I think it would make it clearer what we're checking for.